modules/labs-react/drawer/spec/Drawer.spec.tsx

```diff
@@ -20,7 +16,7 @@ describe('Drawer', () => {
   });
 
   test('should not render a close icon button', async () => {
-    const {container} = render(<DrawerHeader headerText={'Header Title'} />);
+    const {container} = render(<DrawerHeader title={'Header Title'} />);
```

Comment:
I fixed type checking in spec files in #958, but I didn't add type checking to the CLI. So type errors in spec files were only visible in an IDE. I've added a spec file typecheck to CI now, so these types of type errors will now be caught.